ex1_ch1.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: ask user for input on bill amount
+bill_amount = float(input("How much was your bill? "))
+
+#step2: ask user for input on tip percentage
+tip_percentage = int(input("How much will you tip? "))
+
+#step3: convert tip to a decimal
+tip_amount = tip_percentage/100 * bill_amount
```

Comment:
Excellent here making this a new variable! One nit-pick is to have separation on the division operator:

```
tip_amount = tip_percentage / 100 * bill_amount
```